Header header

float32 horizontal_fov
float32 vertical_fov

uint16 horizontal_samples
uint16 vertical_samples

float32 min_range
float32 max_range

float32[] ranges # 1D array of ranges in meters